documentation/src/main/asciidoc/topics/proc_configuring_sql_stores_query.adoc

```diff
@@ -18,68 +28,61 @@ include::dependencies_maven/persistence_sql_store.xml[]
 
 .Procedure
 
-. Create a SQL cache store configuration in one of the following ways:
-+
-* Declaratively, add the `persistence` element or field then add `query-jdbc-store` with the following schema namespace:
+. Create a cache store configuration that uses SQL queries.
 +
-[source,xml,options="nowrap",subs=attributes+]
+.Declarative
+[source,xml,options="nowrap",subs=attributes+,role="primary"]
 ----
-xmlns="urn:infinispan:config:store:sql:{schemaversion}"
+query-jdbc-store xmlns="urn:infinispan:config:store:jdbc:{schemaversion}"
 ----
 +
-* Programmatically, add the following methods to your `ConfigurationBuilder`:
-+
-[source,xml,options="nowrap",subs=attributes+]
+.Programmatic
+[source,xml,options="nowrap",subs=attributes+,role="secondary"]
 ----
 persistence().addStore(QueriesJdbcStoreConfigurationBuilder.class)
 ----
 
-. Specify the dialect of the database with either the `dialect` attribute or the `dialect()` method.
-. Configure any properties for the SQL cache store as appropriate.
-+
-For example, specify if the cache store is shared between multiple cache instances with either the `shared` attribute or the `shared()` method.
+. Specify the database dialect with either `dialect=""` or `dialect()`, for example `dialect="H2"` or `dialect="postgres"`.
+. Configure the SQL cache store with the properties you require, for example:
 +
-If you want your SQL queries to update and delete entries, add the `read-only="false"` attribute or the `.ignoreModifications(false)` method to allow modifications to persistent storage.
+* To use the same cache store with multiple caches, set `shared="true"` or `shared(true)`.
 +
-. Add a JDBC connection factory so that {brandname} can connect to the database.
-. Specify the SQL query for loading data with either the `queries` element or the `queries()` method.
-.. Add `SELECT`, `SELECT ALL`, and `SIZE` clauses to your query.
-.. Add `DELETE`, `DELETE ALL`, and `UPSERT` clauses if the cache store allows modifications.
+* To create a read only cache store, set `read-only="true"` or `.ignoreModifications(true)`.
+
+. Define SQL query statements that load caches with data and modify database tables with the `queries` element or the `queries()` method.
```

Comment:
May want to mention that if the store is configured as read only the delete and upsert statements are not required.